### ISSUE TITLE
Nan

### ISSUE THREAD

**Issue by pakchoi-php** [Justin900429/PyTorch-DSSN-MER#2]
![image](https://user-images.githubusercontent.com/51792701/155095031-b717ef14-8616-4734-941e-0f2179eb0010.png)

Hello, I encountered the problem in the figure when running your reproduction code. I don't know how to solve it. In addition, num_ Class = 5 reports an error. Have you modified the casme dataset table?

**Comment by Justin900429** [comment]
I have released the new preprocessing code, you can try again.

If the same thing still happens, I will test the code and release the new version. Tks! :)

**Comment by pakchoi-php** [comment]
> I have released the new preprocessing code, you can try again.
> 
> If the same thing still happens, I will test the code and release the new version. Tks! :)

I run it with your updated code. I don't know what the problem is. Can you run it again? My code ability is a little poor, and your comments are very clear, so I ran your code and tried it. There is also a problem with the accuracy of your Graph learning before. I don't know whether you will update it again.

**Comment by Justin900429** [comment]
I will test it, but need some time. 

Do you mean the FMER reimplementation?
I have to say the accuracy is really poor and I have no idea why it don't work XD

**Comment by Justin900429** [comment]
Hi @pakchoi-php, I finished modifying the codes and rewriting the README file.
You can reclone the project and test it again!

**Comment by pakchoi-php** [comment]
Hello, is your accuracy up to the value in the paper? PartialAlexNet pre-training, I used lrn-epoch-099.pth, is not the problem here that caused me f1=55%

**Comment by Justin900429** [comment]
The table of the README is outdated. I will remove it to avoid confusing the users.
I have not tested the f1-score for the new code. You can try different combination of the hyper-parameters.
Other promising direction is to check the implementation of optical flow and optical strain.

**Comment by Justin900429** [comment]
I'll close the issue.
Reopen the issue if you want to discuss more.
Thanks!
